### ISSUE TITLE
Whep don't work

### ISSUE THREAD

**Issue by pikachu937** [bluenviron/mediamtx#1883]
## Which version are you using?

v0.23.3

## Which operating system are you using?

- [x] Linux amd64 standard
- [ ] Linux amd64 Docker
- [ ] Linux arm64 standard
- [ ] Linux arm64 Docker
- [ ] Linux arm7 standard
- [ ] Linux arm7 Docker
- [ ] Linux arm6 standard
- [ ] Linux arm6 Docker
- [ ] Windows amd64 standard
- [ ] Windows amd64 Docker (WSL backend)
- [ ] macOS amd64 standard
- [ ] macOS amd64 Docker
- [x] Other (LXC Debian 11)

## Describe the issue

whep not working from external web application or gstreamer
```yaml
webrtcDisable: no
webrtcAddress: :8889
webrtcEncryption: no
webrtcServerKey: server.key
webrtcServerCert: server.crt
webrtcAllowOrigin: '*'
webrtcTrustedProxies: []
webrtcICEServers: [stun:stun.l.google.com:19302]
webrtcICEHostNAT1To1IPs: []
webrtcICEUDPMuxAddress:
webrtcICETCPMuxAddress:
```

but if open https://10.100.100.229:8889/path2/com in browser, all good

## Describe how to replicate the issue

1. start the server
2. publish with rtsp
3. read with webrtc over whep

## Did you attach the server logs?

**yes** / no

log1.txt  - https://10.100.100.229:8889/path2/com/whep
log2.txt  - https://10.100.100.229:8889/path2/com/whep/

[log1.txt](https://github.com/bluenviron/mediamtx/files/11610456/log1.txt)
[log2.txt](https://github.com/bluenviron/mediamtx/files/11610457/log2.txt)

## Did you attach a network dump?

yes / **no**

**Comment by aler9** [comment]
Hello, can you post a sample GStreamer command line that allows to replicate the issue?

**Comment by pikachu937** [comment]
I'm sorry, I didn't have access to github for a while. the problem is solved in version 0.23.6.
